### ISSUE TITLE
Sponsor Badge CSS overflow issue on the docs

### ISSUE THREAD

**Issue by Kludex** [fastapi/fastapi#12239]
### Discussed in https://github.com/fastapi/fastapi/discussions/12218

<div type='discussions-op-text'>

<sup>Originally posted by **nat236919** September 19, 2024</sup>
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
announce-wrapper .sponsor-badge {
    display: block;
    position: absolute;
    top: -10px;
    right: 0;
    font-size: 0.5rem;
    color: #999;
    background-color: #666;
    border-radius: 10px;
    padding: 0 10px;
    z-index: 10;
}
```

### Description

![image](https://github.com/user-attachments/assets/0b9cc528-d268-4e2f-b073-0988228ebd55)

Due to its absolute position and overflow setting, the text is trailing down creating an expected scroll. I think we can simply solve the issue by removing **position: absolute;**

### Operating System

Windows

### FastAPI Version

NA

### Pydantic Version

NA

### Python Version

NA

### Additional Context

_No response_</div>

**Comment by nat236919** [comment]
@Kludex Agreed. Would you like me to work on this? A simple change in .**announce-wrapper .sponsor-badge** CSS class would do.

**Comment by code-yeongyu** [comment]
.

**Comment by arendondiosa** [comment]
Hi. I was able to replicate the error (I see this is an old issue but it is opened 😅 )
<img width="1592" height="633" alt="Image" src="https://github.com/user-attachments/assets/f3133d66-35b4-4330-aaad-0eb35ac41ac3" />
If an Ad blocker extension is enabled, the banner is blocked/hidden. I see a couple of options

1. Search a way to avoid the blocker 
2. Hide the seccion if the Ad blocker is enabled

**Comment by YuriiMotov** [comment]
Should we even care about this?
* As I understand this only happens if (any of):
  * people use AD blockers. They can always add `tiangolo.com` to exceptions and look at our beautifull sponsors, right?
  * the URL of the image is broken - I believe this will be reported and fixed quickly.
  * connection error - this should happen often.
* This doesn't look aweful. I believe people who are using AD blockers are tolerant to such things
* People who are using AD-blockers can just add `tiangolo.com` to the list of exceptions and enjoy our beutifull sponsors :)

**Comment by mlgarchery** [comment]
To be honest, it does look quite bad. 
I am having the issue with Vivaldi (not Brave nor Firefox that also have some adblocking natively).

<img width="772" height="228" alt="Image" src="https://github.com/user-attachments/assets/765a2995-01f4-4874-980a-37b123d318bb" />
